ios/DroidKaigi 2021/DroidKaigi 2021/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Why do we need these files?

Reply:
To show language setting in Setting App. 
If this files are not existed, no settings found in Setting App.

Reply:
Because we have done the language from setting, color theme could be the same I guess.
If we have time to implement this, I think it's okay to keep like this.